lib/ueberauth/strategy/slack/oauth.ex

```diff
@@ -59,7 +63,7 @@ defmodule Ueberauth.Strategy.Slack.OAuth do
   defp endpoint("/" <> _path = endpoint, client), do: client.site <> endpoint
   defp endpoint(endpoint, _client), do: endpoint
 
-  defp to_url(client, endpoint, params \\ nil) do
+  defp to_url(client, endpoint, params) do
```

Comment:
Why did you remove the default value here?

Reply:
Because the following warning:

```
warning: default arguments in to_url/3 are never used
  lib/ueberauth/strategy/slack/oauth.ex:66
```

Which hints to:

```
      if params do
        endpoint <> "?" <> URI.encode_query(params)
      else
        endpoint
      end
```

Basically, we're already checking params - so no need for a default.